client-sdk/ts-web/core/playground/sample-envoy.yaml

```diff
@@ -56,7 +56,7 @@ static_resources:
         - endpoint:
             address:
               pipe:
-                path: node/internal.sock
+                path: /tmp/oasis-net-runner-sdk-core/net-runner/network/validator-0/internal.sock
```

Comment:
Shouldn't this point to a client node?

Reply:
It should. I found out though that by not provisioning a runtime, it doesn't create a client node. So at this time I've switched it over to connect directly to the validator \\:

Reply:
opened https://github.com/oasisprotocol/oasis-core/issues/3794 to ask about this

Reply:
Oh, so there will be a separate test for runtimes, I see.